service/src/models/archaeologist.ts

```diff
@@ -153,29 +153,34 @@ export class Archaeologist {
   }
 
   async publishEnvConfig(connection) {
-    const envConfig = {
-      encryptionPublicKey: this.envConfig.encryptionPublicKey,
-      peerId: this.peerId.toString(),
-    };
-
-    const signature = await this.web3Interface.signer.signMessage(JSON.stringify(envConfig));
-
-    const configStr = JSON.stringify({
-      signature,
-      ...envConfig
-    });
-
-    const { stream } = await connection.newStream(`/env-config`);
-
-    pipe(
-      [new TextEncoder().encode(configStr)],
-      stream,
-      async (source) => {
-        for await (const data of source) {
-          const dataStr = new TextDecoder().decode(data as BufferSource | undefined);
-          console.log('dataStr', dataStr);
+    try {
+      const envConfig = {
+        encryptionPublicKey: this.envConfig.encryptionPublicKey,
+        peerId: this.peerId.toString(),
+      };
```

Comment:
making a note to followup later to see if we need the peerId here.

The PeerID should be available on the connection object the webapp has when the connection is made, so I don't think it needs to be sent back here.